DESCRIPTION

```diff
@@ -1,22 +1,22 @@
 Package: jaspAudit
 Type: Package
 Title: JASP for Audit
-Version: 0.17.2
-Date: 2023-02-08
+Version: 0.17.3
+Date: 2023-06-29
 Author: Koen Derks, Jacques de Swart, Eric-Jan Wagenmakers, Jan Wille, and Ruud Wetzels
 Website: https://github.com/jasp-stats/jaspAudit
 Maintainer: Koen Derks <k.derks@nyenrode.nl>
 Description: Statistical methods for auditing.
-License: GPL (>= 2)
+License: GPL (>= 3)
```

Comment:
why this change? this should be standardized within JASP.

Reply:
It contradicted this line in `description.qml`:

```
license				: "GPL (>= 3)"
```